contracts/src/CrossChainApplications/NativeTokenBridge/INativeTokenDestination.sol

```diff
@@ -0,0 +1,39 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+/**
+ * @dev Interface that describes functionalities for a cross-chain native token bridge.
+ */
+interface INativeTokenDestination {
+    /**
+     * @dev Emitted when tokens are burned in this bridge contract to be bridged to the source chain.
+     */
+    event TransferToSource(
+        address indexed tokenContractAddress,
+        uint256 indexed teleporterMessageID,
+        bytes32 destinationChainID,
+        address destinationBridgeAddress,
+        address recipient,
+        uint256 transferAmount,
+        uint256 feeAmount
+    );
```

Comment:
```suggestion
    event TransferToSource(
        address indexed sender,
        address indexed recipient,
        uint256 amount,
        address feeContractAddress        
        uint256 feeAmount,
        uint256 teleporterMessageID
    );
```

Is the `tokenContractAddress` meant to be the `feeTokenContractAddress`? If so, I'd recommend moving it to be the second to last argument here, just before `feeAmount`. I don't think it needs to be indexed either. 

I'd also recommend removing the `destinationChainID` and `destinationBridgeAddress` here, since they will be same for every event emitted from the given contract.